src/main/java/edu/berkeley/ground/dao/models/cassandra/CassandraTagFactory.java

```diff
@@ -76,16 +78,17 @@ public CassandraTagFactory(CassandraClient dbClient) {
     return result;
   }
 
-  @Override
   public List<Long> getVersionIdsByTag(String tag) throws GroundException {
-    return this.getIdsByTag(tag, "rich_version");
+    return ElasticSearch.getSearchResponse("rich_version", tag);
```

Comment:
We should probably have a configuration option that allows people to turn on / off ElasticSearch use. Let's add a field to the config and pass it through to the `TagFactory`. If it's turned off, we can go to the regular database (current impl.), and if it's turned on, we can use the ElasticSearch API.